Sources/Bot/Services/MergeService.swift

```diff
@@ -390,8 +394,11 @@ extension MergeService {
 
         return Feedback(skippingRepeated: { $0.status.integrationMetadata }) { metadata -> SignalProducer<Event, Never> in
 
-            guard metadata.isMerged == false
-                else { return .value(.integrationDidChangeStatus(.done, metadata)) }
+            guard metadata.isMerged == false else {
+                return SignalProducer
```

Comment:
The compiler was complaining about this?

Reply:
It's because I added a missing `observe(on:)` to the chain. So now we can't use the shorthand dot-leading syntax since it's a chain and not a single expression